index.js

```diff
@@ -285,7 +292,7 @@ ExtractTextPlugin.prototype.apply = function(compiler) {
 				callback();
 			}.bind(this));
 		}.bind(this));
-		compilation.plugin("additional-assets", function(callback) {
+		compilation.plugin("before-chunk-assets", function() {
```

Comment:
Maybe the question is what's worth testing. It could be possible there is no good way just yet.